src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -131,6 +139,10 @@ public void executionPhaseFinish(@SuppressWarnings("unused") ExecutionFinishedEv
   @AllowConcurrentEvents
   public void actionResultReceived(ActionResultReceivedEvent event) {
     spawnStats.countActionResult(event.getActionResult());
+    Optional<Duration> cpuUserTimeForActionsDuration = event.getActionResult().cumulativeCommandExecutionUserTime();
```

Comment:
Nit: You don't need to have `Duration` at the end of this variable name, that's obvious from the type. In fact, since the variable is so short-lived, use a shorter variable name, or just inline it.